stylus/base/palette.styl

```diff
@@ -18,9 +18,13 @@ grey-05 = #92A0B2
 grey-06 = #748192
 grey-07 = #4F5B69
 grey-08 = #32363F
+grey-09 = #d6d8da
+grey-10 = #f5f6f7
+grey-11 = #95999d
 
 // more of cozy UI
 blue      = #33A6FF
+blue2     = #297ef2
```

Comment:
Yeah… i almost overwrote the blue color with the new one but i thought that maybe it was still in use in another app. Then i thought that i could use [Name that color](http://chir.ag/projects/name-that-color/). Really i don't know for the time being what's the most clever choice.